3scale/APIcast
# Bump penlight from 1.7.0 to 1.31.1

### What

This commit also removes filesystem.lua as the issue has been fixed in penlight v1.8.0

https://github.com/lunarmodules/Penlight/pull/329

The dev image is already run with version 1.13.1 
https://github.com/3scale/APIcast/blob/master/gateway/Roverfile.lock#L29

### Verification Steps

* Build docker image from this git branch
```
make runtime-image IMAGE_NAME=apicast-test
```
* HTTP dev environment setup
```
cd dev-environments/plain-http-upstream
make gateway IMAGE_NAME=apicast-test
```
The gateway should be running without error
* The request should be accepted (200 OK)
```
curl --resolve get.example.com:8080:127.0.0.1 -v "http://get.example.com:8080/?user_key=123"
```
